ontolearn/triple_store.py

```
@@ -1195,4 +1195,16 @@ def query(self, sparql: str):
         yield from self.g.query(sparql_query=sparql)
 
     def query_results(self, sparql: str):
-        return self.g.query(sparql_query=sparql)
+        trials = 0
+        response = None
+        while True:
```

There is no need this while loop. If the connection is not in place or timeout is not specified for the request, then we should throw error instead of returning None. Therefore, please remove your changes.